tests/POW/test_POW.cpp

```diff
@@ -740,6 +740,96 @@ BOOST_AUTO_TEST_CASE(mining_and_verification)
     BOOST_REQUIRE(!verifyWinningNonce);
 }
 
+BOOST_AUTO_TEST_CASE(opencl_gpu_mining_and_verification_1)
+{
+    if (!OPENCL_GPU_MINE)
+        return;
```

Comment:
enclosed in {}

Reply:
Hi Jun Hao, do you mean we should enclode the return; statement in {}?

Reply:
typically, as a good practice, we use opening/closing braces for the if statement. This is to avoid mistake when the code is updated. 

https://github.com/Zilliqa/Zilliqa/wiki/Coding-Guidelines